CMakeLists.txt

```diff
@@ -190,6 +190,7 @@ endif()
 if (APPLE)
     # build macOS File Provider module
     option(BUILD_FILE_PROVIDER_MODULE "BUILD_FILE_PROVIDER_MODULE" ON)
+    add_definitions(-DBUILD_FILE_PROVIDER_MODULE=${BUILD_FILE_PROVIDER_MODULE})
```

Comment:
please put the definition in `config.h.in`

Reply:
Done

Reply:
but if you put it in `config.h.in` then the explicit define is no longer needed or do I miss something ?